docs/specification.md

```diff
@@ -610,6 +601,7 @@ Parameter (in Michelson):
   - If proposal got accepted:
     - The return amount for the proposer is equal to or less than the sum of the proposer frozen tokens and the fee paid for the proposal.
     - The return amount for each voters is equal to or less than the voter's frozen tokens.
+  - The token return to voters are not immediate. The voters should call `unstake_vote` to get their tokens after `flush` is called.
```

Comment:
We should also probably make it clear that voters should hold on to the proposal keys that they have voted on, if they want their staked tokens back.

Reply:
They could also use an indexer/explorer for this tho.

For example, it would be sufficient to look at one's previous call to the `vote` entrypoint or traversing thie keys of the votes big_map.

In general I would avoid making suggestions in the specification when possible, given that this is supposed to be a document strictly describing what the contract does and how it behaves.

Reply:
Ok, I think we can at least say "should call `unstake_vote` with the proposal keys"

Reply:
I've added `with the proposal keys` part.